covariancematrix.go

```diff
@@ -25,8 +26,6 @@ func CovarianceMatrix(cov *mat64.Dense, x mat64.Matrix, wts []float64) *mat64.De
 
 	r, c := x.Dims()
 
-	// TODO(jonlawlor): indicate that the resulting matrix is symmetric, which
```

Comment:
Why is this TODO being removed? We probably want this to return *mat64.Symmetric when that happens.